frontend/src/components/program/picasso/Picasso.vue

```diff
@@ -103,8 +102,7 @@ Listing all given activity schedule entries in a calendar view.
 
           <schedule-entry-responsibles
             :schedule-entry="event"
-            :avatar-size="20"
-            class="d-flex justify-end flex-wrap mt-2"
+            :avatar-size="event.duration < 30 ? 12 : event.duration < 60 ? 16 : 20"
```

Comment:
Not consistent between read and edit mode. Here it is `<30`, above it is `<=30`.
Additionally: I find 12 to be very small. On my screen, barely readable.

Reply:
The problem is, that when the event is less or about 30minutes. The size can't be bigger anyway, if you want to see the avatar in several combinations, because the event height is no more than 18px anyway. I could make the size instead dependent on the actual interval height.

Reply:
> I could make the size instead dependent on the actual interval height.

If this is possible, that would be great to make it adjustable to different screens.
On my screen it currently looks like the screenshot below for a 30min event. The scheduleEntry div ist 27px high.

<img width="309" alt="image" src="https://user-images.githubusercontent.com/449555/190838530-6fe905cd-8e53-4e49-952c-fa16a6193e73.png">

I also noticed, that in edit mode it collides now with the pencil icon.